chainntnfs/bitcoindnotify/bitcoind.go

```diff
@@ -828,7 +850,8 @@ func (b *BitcoindNotifier) RegisterSpendNtfn(outpoint *wire.OutPoint,
 // script. If one is found, it's spending details are sent to the TxNotifier,
 // which will then dispatch the notification to all of its clients.
 func (b *BitcoindNotifier) dispatchSpendDetailsManually(
-	historicalDispatchDetails *chainntnfs.HistoricalSpendDispatch) error {
+	historicalDispatchDetails *chainntnfs.HistoricalSpendDispatch) (
```

Comment:
Should rename the method and update the `godoc` as it's no longer dispatching the details.

Reply:
fixed